cmd/init.go

```diff
@@ -103,10 +105,15 @@ dapr init --from-dir <path-to-directory>
 				dockerNetwork = viper.GetString("network")
 				imageRegistryURL = viper.GetString("image-registry")
 			}
-			if fromDir != "" {
+			// If both --image-registry and --from-dir flags are given, error out saying only one can be given.
+			if len(strings.TrimSpace(imageRegistryURL)) != 0 && len(strings.TrimSpace(fromDir)) != 0 {
```

Comment:
Is `TrimSpace` needed here? It's directly coming from the viper/pflag.

Reply:
```bash
$ ./release/dapr init --from-dir "   " --image-registry localhost:5000  
⌛  Making the jump to hyperspace...
⚠  Flag --image-registry is a preview feature and is subject to change. It is only available from CLI version 1.7 onwards.
❌  could not connect to Docker. Docker may not be installed or running
```

I had local docker turned off... but if "   " is given within quotes, viper/pflag still allows it through as a value ...
But in this case if `TrimSpace` is not there 
```bash
./release/dapr init --from-dir "   " --image-registry localhost:5000 
⌛  Making the jump to hyperspace...
❌  both --image-registry and --from-dir flags cannot be given at the same time
```
This should not be the case since `--from-dir` is empty and has no valid value.

Reply:
Hmm, if user is giving an invalid value (they are intentional about with since they are using quotes), shouldn't it error out?

Reply:
IMO no it should not.

Reply:
current approach looks good to me.